docs/install/lin/activating-a-validator.md

```diff
@@ -135,16 +151,16 @@ docker run -it -v $HOME/prysm/validator:/data --network="host" \
 bazel run //validator -- --keymanager=keystore --keymanageropts=${HOME}/prysm/validator/keystore.json
 ```
 
-## Submitting the deposit contract
+## Step 5: Submitting the deposit contract
 
 Once both the beacon node and validator client are successfully running, make your deposit by clicking the button and following the steps presented in your wallet.
 
 ![](https://blobscdn.gitbook.com/v0/b/gitbook-28427.appspot.com/o/assets%2F-LRNnKRqTm4z1mzdDqDF%2F-LuJpxGKxOpat8TfDxPP%2F-Lua3RjIGSbGQbe7NrjZ%2F5.png?alt=media&token=0561a974-edf7-49f9-b225-8997982eb8e0)
 
-It will take a while for the nodes in the network to process a deposit. Once the node is active, the validator will immediately begin performing its responsibilities.
+## Step 6: Wait for your validator assignment
 
-The validator is now awaiting its first assignment from the network. This should only take a few minutes, and an indication will be displayed on both the prylabs.net page as well as the validator client.
+It will take a significant while for the nodes in the network to process a deposit. Meanwhile, leave both terminal windows open and running. Once the node is actived by the ETH2 network, the validator will immediately begin performing its responsibilities. The validator is now awaiting its first assignment from the network.
```

Comment:
This paragraph doesn't make much sense and has a couple typos. It should be something along the lines of:

 > Please note that it may take up to 12 hours for nodes in the network to process a deposit. In the meantime, leave both terminal windows open and running; once the node is activated by the ETH2 network, the validator will immediately begin receiving tasks and performing its responsibilities.

Reply:
Thanks,  That's much better.